crates/node-core/src/args/rpc_server.rs

```diff
@@ -469,196 +323,3 @@ impl TypedValueParser for RpcModuleSelectionValueParser {
         Some(Box::new(values))
     }
 }
-
-#[cfg(test)]
-mod tests {
```

Comment:
I want to keep these tests but no longer make use of the trait functions

Reply:
For instance replacing:
```rust
let config = args.transport_rpc_module_config();
```
by something like:
```rust
let config = TransportRpcModuleConfig::default();
```
implies moving `TransportRpcModuleConfig` and many others config types to rpc-server-types. Is that what we want?

Reply:
this test can be moved to where we impl serverconfig trait, so I guess we need to move some tests and keep some